### PR TITLE
ISO8601 upgrade (0.2.0).

### PR DESCRIPTION
**Pull request by sagotch**

### CODE REVIEW

File: packages/ISO8601/ISO8601.0.2.0/opam [hunk 1]
```diff
@@ -0,0 +1,29 @@
+opam-version: "1.2"
+
```
Comment (Chris00) [inline packages/ISO8601/ISO8601.0.2.0/opam]: Why all these empty lines?

Comment (sagotch) [inline packages/ISO8601/ISO8601.0.2.0/opam]: I like spaces. A lot :+1: I just find it more readable.

**Comment by sagotch** [comment]
One more time, build failed because of a timeout while initializing: https://travis-ci.org/ocaml/opam-repository/jobs/52942063

Could you please relaunch the build?

**Comment by yallop** [comment]
I've restarted the build.

**Pull request merged**